client/src/Components/ItemsForms.jsx

```diff
@@ -0,0 +1,40 @@
+export function ItemForms({ numberOfItems, out }) {
+
+  const forms = [];
```

Comment:
You aren't really creating forms here. A form is a collection of input elements. Instead, I think these are all items. Your loop control "numberOfItems" seems to confirm that.